openslides/motion/forms.py

```diff
@@ -49,6 +49,11 @@ class BaseMotionForm(CleanHtmlFormMixin, CssClassMixin, forms.ModelForm):
     Attachments of the motion.
     """
 
+    tags = forms.ModelMultipleChoiceField(
+        queryset=Tag.objects.all(),
+        required=False,
+        label=ugettext_lazy('Tags'))
+
```

Comment:
Yes. The motion form is generated automaticly from different Form-Classes. So it does not work the "normal" way